plugins/woocommerce-admin/client/dashboard/components/connect/index.js

```diff
@@ -155,22 +147,23 @@ Connect.defaultProps = {
 	setIsPending: () => {},
 };
 
+const formatJetpackAuthUrlQueryArgs = ( redirectUrl ) => ( {
```

Comment:
Extracted this here so that both the Connect component and the PrefetchJetpackAuthUrl construct the query args the same way (required for prefetching to work correctly).